app/views/posts/show.html.erb

```diff
@@ -2,12 +2,15 @@
 <html>
 <head>
   <title>Post Details</title>
+  <link rel="stylesheet" type="text/css" href="/assets/stylesheets/posts.css">
 </head>
 <body>
   <h1>Post Details</h1>
-  <h2><%= @post.title %></h2>
-  <p>Author: <%= @post.author.name %></p>
-  <p>Created At: <%= @post.created_at.strftime('%B %d, %Y') %></p>
-  <p><%= @post.text %></p>
+  <div class="post-details">
+    <h2><%= @post.title %></h2>
+    <p class="post-info">Author: <%= @post.author.name %></p>
+    <p class="post-info">Created At: <%= @post.created_at.strftime('%B %d, %Y') %></p>
+    <p class="post-text"><%= @post.text %></p>
+  </div>
 </body>
 </html>
```

Comment:
- [ ] Kindly improve the post show page to follow the required layout.

  ![image](https://github.com/mpa-mxiang/Blog-app/assets/81468675/943d8544-f1cb-400a-b909-3d7287638ff4)

  ![image](https://github.com/mpa-mxiang/Blog-app/assets/81468675/c90e7db0-1180-4f70-9633-e0bd2faf9f62)